src/scripts/background.js

```diff
@@ -286,28 +295,28 @@ window.TogglButton = {
       type: 'pong'
     });
 
-    TogglButton.$socket.onopen = function () {
+    TogglButton.websocket.socket.onopen = function () {
+      TogglButton.resetWebsocketState();
```

Comment:
My only suggestion would be some alternate name, this stood out to me since I thought it resets the actual websocket connection as soon the socket connection is opened

Maybe `resetWebsocketRetryCount`  or something